content/docs/guides/troubleshooting/control_plane_error_codes.md

```diff
@@ -1,11 +1,12 @@
 ---
-title: "Error Code Troubleshooting"
-description: "OSM Control Plane Error Code Troubleshooting Guide"
+title: "Error Codes"
+description: "Troubleshooting control plane error codes"
 aliases: ["/docs/troubleshooting/observability/errorcodes/"]
 type: docs
+weight: 5
 ---
 
-# OSM Control Plane Error Code Troubleshooting Guide
+# OSM Control Plane Error Codes
```

Comment:
This doesn't quite follow the pattern but I'm fine with that :)

Reply:
Updated this to `Troubleshooting OSM Control Plane Error Codes`.